shared/chat/conversation/info-panel/index.js

```diff
@@ -25,7 +25,7 @@ const listStyle = {
     : {
         backgroundColor: globalColors.white,
         borderLeft: border,
-        marginTop: -4 /* Necessary fix: adds 1px at the top so we hide the gray divider */,
+        marginTop: -5 /* Necessary fix: adds 1px at the top so we hide the gray divider */,
```

Comment:
I think this style should be removed, `clickCatcherStyle` in `info-panel/container` moves the panel down and this is moving it back up again. `clickCatcherStyle` can just not move it down as much.